qa/workunits/windows/run-tests.ps1

```diff
@@ -0,0 +1,29 @@
+$ProgressPreference = "SilentlyContinue"
+$ErrorActionPreference = "Stop"
+
+$scriptLocation = [System.IO.Path]::GetDirectoryName(
+    $myInvocation.MyCommand.Definition)
+
+$testRbdWnbd = "$scriptLocation/test_rbd_wnbd.py"
+
+function safe_exec() {
+    # Powershell doesn't check the command exit code, we'll need to
+    # do it ourselves. Also, in case of remote commands, it treats stderr
```

Comment:
By remote do you mean a regular `.exe` executed locally (i.e. not a PowerShell cmdlet)?  I was confused here for a second looking for something resembling `ssh` around ;)

Reply:
> This script is invoked through ssh: https://github.com/ceph/ceph-build/blob/main/scripts/ceph-windows/run_tests#L69.

I know.  But `safe_exec` is about invoking something that is not PowerShell cmdlet locally, right?

> 
> Powershell has some inconsistencies. When running a .ps1 script locally, it may ignore stderr output. However, when invoked through ssh or winrm, it treats it as an exception, even if the return code is 0. They recently introduced an experimental setting for this: https://learn.microsoft.com/en-us/powershell/scripting/learn/experimental-features?view=powershell-7.3#psnativecommanderroractionpreference

I'd change "remote" to "native" in this comment.

Reply:
sure, updated